catalog/app/containers/Admin/Table/Table.tsx

```diff
@@ -186,36 +208,25 @@ export function InlineActions({ actions = [], children = undefined, ...props })
   )
 }
 
+const useWrapperStyles = M.makeStyles({
+  root: {
+    overflowX: 'auto',
+  },
+})
+
+export function Wrapper({ className = undefined, ...props }) {
```

Comment:
doesn't TS complain about missing type annotations?

Reply:
No. I think it's because `className` has inferred `undefined` type, and destructured object - `Record<string, any>`

I added this:
```tsx
+export function Wrapper({
+  className = undefined,
+  ...props
+}: React.HTMLAttributes<HTMLDivElement>) {
+  const classes = useWrapperStyles()
+  return <div className={cx(classes.root, className)} {...props} />
+}
```